common/data_refinery_common/message_queue.py

```diff
@@ -33,6 +35,196 @@
 BATCH_TRANSCRIPTOME_JOB = "TRANSCRIPTOME_INDEX"
 BATCH_DOWNLOADER_JOB = "DOWNLOADER"
 NONE_JOB_ERROR_TEMPLATE = "send_job was called with NONE job_type: {} for {} job {}"
+TIME_OF_LAST_JOB_CHECK = timezone.now() - datetime.timedelta(minutes=10)
+
+
+batch = boto3.client("batch", region_name=AWS_REGION)
+
+# Initialize the queue depths to zero to be used as a global.
+JOB_QUEUE_DEPTHS = {}
+for queue_name in settings.AWS_BATCH_QUEUE_ALL_NAMES:
+    JOB_QUEUE_DEPTHS[queue_name] = 0
+
+DOWNLOADER_JOB_QUEUE_DEPTHS = {}
+for queue_name in settings.AWS_BATCH_QUEUE_WORKERS_NAMES:
+    DOWNLOADER_JOB_QUEUE_DEPTHS[queue_name] = 0
+
+
+def count_jobs_in_queue(batch_job_queue) -> int:
```

Comment:
would it make more sense to have the name reflect the return value(s)

Reply:
The type declaration is also wrong, because this function returns a pair of ints